castervoice/rules/apps/editor/unity.py

```diff
@@ -0,0 +1,64 @@
+from dragonfly import Key,MappingRule, Dictation, Choice,Text
+from castervoice.lib.merge.state.short import R
+from castervoice.lib.merge.additions import IntegerRef
+from castervoice.lib.ctrl.mgr.rule_details import RuleDetails
+
+class UnityRule(MappingRule):
+	mapping = {
+		"<window>" : R(Key("c-%(window)s")),
+		"search <searchable> [<dictation>]" : R(Key("c-%(searchable)s,c-f") + Text("%(dictation)s")),
```

Comment:
Wow, had no idea a command like this existed!

Reply:
It doesn't in Unity. It's just a series of key strokes compressed to one command. For Caster commands in general I love the ones that let me add dictation without a break. And there's a lot of potential for creating more commands in Unity for any actions that take the same input every time.